Chapters/4-6-Chapter28.md

```diff
@@ -0,0 +1,201 @@
+
+"8 Tova'la left," I said to myself, recalling the times Jathi had confirmed.
+
+"You are not forgotten."
+
+I looked back at Laen sleeping face-to-face with Boon, wondered what they felt when they thought about Liffa or Pomri, if explaining it to them would be possible, or if it would even help. Maybe they simply trusted us to make things good in the end. I couldn't bear to tell them the truth.
+
+With quiet steps I walked outside, being careful not to wander too far from cove. There was nary a sign of the scuffle that Annu had saved me from. The pebbles had been shuffled and the bodies reclaimed by the many fingers of the ocean. I searched the shoreline for the place where Pomri had been taken, but couldn't find it. The water had erased any evidence of its happening, without which virtue could hold no contempt.
+
+"Good Morning, Linyu," called Annu from afar.
+
+"You're walking now," I said in amazement, and noticed that thi had a caution with which thi took hae steps.
+
+"Only just," thi said nonchalantly, "My eyes are taking on the blue again, and I can't sleep."
+
+"The visions?"
+
+Thi nodded. As thi walked closer, I could see the faint speckles growing in magnitude and number. I understood then how thi came to never die. That kanosin seemed to be growing inside of hae somehow. I came to hae and gestured on hae skin toward the kelp bandage on hae wound.
+
+"...May I?" I asked.
+
+Thi nodded, understanding my curiosity was for hae health. The bandage stuck well to hae skin, but I managed to peel a bit of it and saw only a mild scar. A swell of coruscating blue dots lie directly underneath.
+
+"Well, go on," thi said, gesturing to the bandage. Slowly, I peeled it away to reveal the entire scar. Somehow, one could tell that it had healed quickly; the skin seemed to grow at different rates, leaving a misshapen, jagged crevasse of young hydroderm. Around the old scar were older ones, some small, some large. There were far more than I could count in the moment before thi closed hae robes.
+
+"Come now. Let's get you back to Ulukoma," thi said.
+
+"We need a ship," I noted. "For the saffas."
+
+"A ship would be difficult to find and even harder to steal. Two canoes should suffice."
+
+I thought thi was joking at first, but a glance showed thi was serious.
+
+"A saffa in a canoe?"
+
+"I've seen them in the water. They could swim the whole way to Ulukoma if the water wasn't so cold."
+
+I imagined Boon and Laen swimming, their fuzzy heads jutting above the waves and wide paws pushing against the current. It looked so strange in my head, but I conceded that two canoes might be our only option.
+
+"Rudjan," thi said, "That's the only place we'll be able to find them."
+
+I imagined the impenetrable walls of Rudjan. How they filled out into the ocean to divide the throngia from the saffas and tilmarims; at least that's what Pomri had told me.
```

Comment:
What are throngia and tilmarims? It's too late into the story to be introducing new things.

Reply:
I think part of the reason why I say this is typically writers set a majority of their world-building by a little past midpoint. This will give the detective-readers all the tools to solve the problem. So the majority of the books will normally never introduce new things this late in the story. This is why users feel cheated with deus ex machina solutions because it added an extra player they didn't account for when they gather their tools to solve the problem.

This might not be relevant to what you are doing here with these new worlds. But it definitely raised alarm bells instinctually for me because we had been delivered stories to us that when it breaks those rules, it stands out. Most readers will just think it feels wrong, and they can't put their finger on why.